test/Microsoft.ML.Benchmarks/KMeansAndLogisticRegressionBench.cs

```diff
@@ -46,13 +46,7 @@ public ParameterMixingCalibratedPredictor TrainKMeansAndLR()
                         }
                     }, new MultiFileSource(_dataPath));
 
-                IDataTransform trans = CategoricalTransform.Create(env, new CategoricalTransform.Arguments
-                {
-                    Column = new[]
-                    {
-                        new CategoricalTransform.Column { Name = "CatFeatures", Source = "CatFeatures" }
-                    }
-                }, loader);
+                IDataView trans = CategoricalTransform.Create(env, loader, new CategoricalEstimator.ColumnInfo("CatFeatures", "CatFeaures"));
```

Comment:
>CategoricalTransform [](start = 34, length = 20)

can we have a convenience ctor here that takes params string[] columns or something?